hackclub.com.yaml

```diff
@@ -332,7 +332,11 @@ alumni:
 angelhacks:
   - ttl: 1
     type: CNAME
-    value: cname.vercel-dns.com.
+    value: cname.vercel-dns.com.  
+ansh:
+  - ttl: 1
+    type: CNAME
+    value: ansh3108.github.io/website/.
```

Comment:
I believe this is not a valid CNAME entry

Reply:
[This guide](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site) should help you set a correct one up